### ISSUE TITLE
Unable to fully create and register for a new acct and associate a device

### ISSUE THREAD

**Issue by pdehaan** [mozilla-services/FindMyDevice#105]
Found in http://fmd.dev.mozaws.net/status/ (commit 0eb7c5aaa38c1418dac4c298cbdc726513d0a9d2)

``` json
{
  "goroutines": 22,
  "status": "ok",
  "version": "0.7-0eb7c5aaa38c1418dac4c298cbdc726513d0a9d2"
}
```

---
### Steps to reproduce:
1. Create a new account via oauth stuff at http://fmd.dev.mozaws.net/
2. Verify the account from step 1.
### Actual results:

I'm on https://oauth-ui.dev.lcip.org/confirm?client_id=344e8ade775e8b15&scope=profile%3Aemail&state=1b08ecb8404fd5a0802309c08de841bb but the verification email uses the following link:

https://accounts.firefox.com/signup_complete?uid=046ca49593c9435aaa1d4140de18e205&code=a541204944a808fd74ada6f3f1e0efbe&service=344e8ade775e8b15

Things get weirder when the verify link takes you to a blank page which dumps the following in the developer console:

> Failed to load resource: net::ERR_NAME_NOT_RESOLVED https://oauth.firefox.com/v1/client/344e8ade775e8b15

But the original tab refreshes to say "You are now ready to use **Find My Device**"
### Expected results:

Less console errors. Plus its a weird jump betweenst the https://oauth-ui.dev.lcip.org to https://accounts.firefox.com somewhere between creating an account and verifying said account (compounded by the fact we don't have a prod oauth endpoint yet, hence _boom_).
Crap hits the fan harder when we have **TWO** tabs in play and one fails and the other claims to succeed, telling me that "You are now ready to use Find My Device" when in fact the oauth callback never succeeded (since it was trying to call missing oauth endpoint) and the account is never created in the database by JR's Go code.

TL;DR: no bueno.
